phpcs.xml.dist

```diff
@@ -134,6 +134,9 @@
 	<exclude-pattern>/src/wp-includes/sodium_compat/*</exclude-pattern>
 	<exclude-pattern>/src/wp-includes/Text/*</exclude-pattern>
 
+	<!-- Compiled files. -->
+	<exclude-pattern>/src/wp-includes/blocks/blocks-json.php</exclude-pattern>
```

Comment:
Is this still needed if the whole `wp-includes/blocks/` directory is included above in [line 126](https://core.trac.wordpress.org/browser/tags/6.0.2/phpcs.xml.dist?marks=126#L124)?

Reply:
Let me try to remove it and check if tests pass. You're right, it should not be necessary 👍

Reply:
Done.